auditlog_ext_nrq/models/auditlog_log.py

```diff
@@ -6,28 +6,33 @@
 
 
 class AuditlogLog(models.Model):
-    _inherit = 'auditlog.log'
+    _inherit = "auditlog.log"
 
-    log_category = fields.Selection([
-        ('state_update', 'Submitted/Set back to Draft'),
-        ('fields_update', 'Fields Update'),
-        ('create', 'Record Created'),
-        ('unlink', 'Record Unlinked'),
-        ('other', 'N/A')],
-        string='Log Category',
-        compute='_get_log_category',
+    log_category = fields.Selection(
+        [
+            ("state_update", "Submitted/Set back to Draft"),
+            ("fields_update", "Fields Update"),
+            ("create", "Record Created"),
+            ("unlink", "Record Unlinked"),
+            ("other", "N/A"),
+        ],
+        string="Log Category",
+        compute="_compute_get_log_category",
```

Comment:
```suggestion
        compute="_compute_log_category",
```

Reply:
@yostashiro I adjusted as you suggested. Thank you for your review.